app/views/contribute.html

```diff
@@ -0,0 +1,186 @@
+{% extends 'layout.html' %}
+{% block pageTitle %}
+  Contribute - NHS prototype kit
+{% endblock %}
+{% block beforeContent %}
+  {{ breadcrumb({ href: "/", text: "Home" }) }}
+{% endblock %}
+{% block content %}
+
+{# the board is a long string to this is made a variable for consistency #}
+{% set board = "https://github.com/orgs/nhsuk/projects/24/views/1" %}
+{# the labels string is also tricky to making a variable #}
+{% set label = "?filterQuery=label%3A" %}
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-two-thirds">
+    <h1 id="contribute-to-the-nhs-prototype-kit">
```

Comment:
Think we should try and always add heading classes to the h1s, to make it clearer whether the intent is for them to be XL or L size? Currently they default to XL but @edwardhorsford and I have previously discussed with folk the idea of either switching the default to L in future or removing the default styling altogether (like GOVUK frontend).

```suggestion
    <h1 class="nhsuk-heading-xl" id="contribute-to-the-nhs-prototype-kit">
```